obs/CMakeLists.txt

```diff
@@ -18,20 +18,27 @@ endif()
 option(ENABLE_BACKTRACE "Enable backtrace support on crash" ON)
 add_feature_info(ENABLE_BACKTRACE ENABLE_BACKTRACE "Backtrace support.")
 
-add_compile_options(
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+set(C_FLAGS
   "-Wall"
   "-Werror"
   "-Wfatal-errors"
   "-ffast-math"
   "-fdata-sections"
   "-ffunction-sections"
-  "-fpic"
   "$<$<CONFIG:DEBUG>:-O0;-g3;-ggdb>"
 )
 
 set(EXE_FLAGS "-Wl,--gc-sections")
 set(CMAKE_C_STANDARD 11)
 
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+	set(OBS_PLUGIN_DIR 64bit)
+else()
+	set(OBS_PLUGIN_DIR 32bit)
```

Comment:
Do we even care about 32-bit, LG would be practically useless on such a system.
Also separate commit please

Reply:
Dunno, but seems better than hard-coding 64bit. Separate commit from what, the `install()` or the cflags?

Reply:
part of the install changes sorry, and yeah, keep this, it's fine :)